target/dovecot/90-sieve.conf

```diff
@@ -108,4 +108,18 @@ plugin {
   # Locations of programs that can be called by the sieve_extprograms plugin
   sieve_pipe_bin_dir = /usr/lib/dovecot/sieve-pipe
   sieve_filter_bin_dir = /usr/lib/dovecot/sieve-filter
+
+  # AntispamWithSieve (Derived from: https://wiki2.dovecot.org/HowTo/AntispamWithSieve)
+
+  # From elsewhere to Spam folder
+  imapsieve_mailbox1_name = Spam
+  imapsieve_mailbox1_causes = COPY
+  imapsieve_mailbox1_before = file:/usr/lib/dovecot/antispam/report-spam.sieve
+
+  # From Spam folder to elsewhere
+  imapsieve_mailbox2_name = *
+  imapsieve_mailbox2_from = Spam
```

Comment:
Should this be Junk?
https://github.com/tomav/docker-mailserver/blob/a7670ac5c122e3ed126dc263b9d23ae3b2997a79/test/config/dovecot-lmtp/conf.d/15-mailboxes.conf#L23